### DISCUSSION TITLE
2주차 스프린트 7/28(목) ~ 7/31(일)

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#4] [Announcements]
# 2차

### 기간

7/28(목) ~ 7/31(일)

### 할 일

**개발 초기 작업**: 스타일 초기화, 마크업 개발, 서버 연동 준비 등

**필수**

- [x]  전역 레이아웃(헤더, 푸터) 마크업 개발 (반응형 포함)
    - 헤더 - 공공, 푸터 - 그린
- [x]  전역 스타일 지정 - 공공
    - 스타일 초기화: reset.css 등
    - Noto Sans KR 로컬에 추가
    - 컬러 상수화
- [x]  SWR, MSW 라이브러리 설치 - 케이
- [x]  각자 맡은 페이지의 마크업 개발 (Antd 활용)
- [x]  깃허브 레포지토리 설정

**추가**

- [x] 더미 데이터 생성 (MSW 활용)
- [ ]  전역 상태 관리, SWR 세팅

**Comment by gitul0515** [comment]
회고
- 목표했던 바를 달성할 수 있어서 팀원들에게 감사하다. 
- 공통 컴포넌트를 미리 파악하고 중복 작업이 없도록 하는 것이 필요하다. 
- Antd 타입 이유 => 해결 
- 반응형이 꽤 어려웠다 
- 레이아웃 일관성 있게 만들기 
- 목업 데이터: 타입 정의가 필요 (이슈 생성)
